scripts/version.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+# Make this script executable from terminal:
+# chmod 755 version.sh
+set -e # Any subsequent(*) commands which fail will cause the shell script to exit immediately
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+LINE="================================================================================"
+VERSION_REGEX="[0-9]+\.[0-9]+\.[0-9]+"
+
+GRADLE_PROPERTIES_FILE=$ROOT_DIR"/code/gradle.properties"
+CONSTANTS_FILE=$ROOT_DIR"/code/target/src/main/java/com/adobe/marketing/mobile/target/Target.java"
```

Comment:
Should be "/code/target/src/phone/java/com/adobe/marketing/mobile/Target.java"?

Reply:
good call